schema/rdl_schema_0.1.json

```
@@ -1025,6 +788,374 @@
 				"Judgement",
 				"Hybrid"
 			]
+		},
+		"Spatial": {
+			"title": "Spatial coverage",
+			"type": "object",
+			"description": "The geographical area covered by the dataset.",
```

I realised that, whilst the property should be named `spatial` to align with DCAT, the definition should be named `Location` since that is what it represents (in DCAT, the `spatial` property references a `Location` class):

```suggestion
		"Location": {
			"title": "Location",
			"type": "object",
			"description": "A geographical area.",
```

---

If you agree, then the `$ref` in `spatial` will need updating, too.

---

I agree so I've updated both of these